sotodlib/preprocess/processes.py

```diff
@@ -1667,9 +1665,7 @@ class FourierFilter(_Preprocess):
 
     Example config file entry for two filters::
 
-        - name: "fourier_filter_chain"
-          wrap_name: "lpf_sig"
-          signal_name: "signal"
```

Comment:
Give one example that passes `signal_name: "demodQ"` and one that passes a wrap name other than the default.